okhttp/src/test/java/okhttp3/KotlinSourceModernTest.kt

```diff
@@ -95,6 +97,11 @@ import org.junit.jupiter.api.Test
 )
 @Disabled
 class KotlinSourceModernTest {
+  @BeforeEach
+  fun disabled() {
```

Comment:
What's this for?

Reply:
@Disabled wasn't working in this setup, it was running it anyway.  And timing out and spewing errors from TODO().  There might be some correct fix, but like the Android 5 junit runner which doesn't support automatic registration of extensions, each new integration has a non zero chance of not handling some feature of test running.